### PR TITLE
Loops: HTML tweaks + test case

### PR DESCRIPTION
**Pull request by mottaquikarim**
- added `<p>` tag around a block of text that was jutting out of content frame
- added example array and expected output to help students solve the final challenge

### CODE REVIEW

File: lecture-loops.html [hunk 1]
```diff
@@ -117,10 +119,13 @@
     <p>
       <b>Challenge Question:</b> Now that we have all the pieces, write a program that iterates through an array from the back to the front and prints out only the values at the odd indices.
     </p>
+
+    <pre><code>var myArray = [1,1,2,3,5,8,13,21,34,55];
+// your program should output: 55,21,13,5,3,1</code></pre>
```
Comment (bensussman) [inline lecture-loops.html]: This are "all the odd values", but the question is "the values at the odd indices"!

I think both questions are valid and interesting. Should we just have both?

**Comment by bensussman** [comment]
I'm not ready to merge this, b/c i don't see why the extra `<p>` is necessary, and the added `<code>` block is actually incorrect! I think maybe we can make it into two questions: one about odd values and one about odd indices, which requires them to know the difference. So let's do that!

**Comment by mottaquikarim** [comment]
Ah yep, that was my bad -- I miscounted one, it should be 55, 21, 8, 3, 1. That being said, this example is still bad now that I think about it because I don't necessarily want all the values at the odd indices to be odd themselves -- this may cause unnecessary confusion. The point of these examples is to make the problem at hand clearer, I will refactor this to include something like [ 3,5,7,9,10,12,14,16 ].

I'll also add an exercise that deals with pulling out the odd values of an array.

**Comment by mottaquikarim** [comment]
@bensussman this should be ready for merge

File: lecture-loops.html [hunk 2]
```diff
@@ -117,10 +119,20 @@
     <p>
       <b>Challenge Question:</b> Now that we have all the pieces, write a program that iterates through an array from the back to the front and prints out only the values at the odd indices.
     </p>
+
+    <pre><code>var myArray = [7, 24, 25, 9, 40, 41, 11, 60, 61];
+// your program should output: 60,41,9,24</code></pre>
+
+    <p>
+    <b>Challenge Question:</b> write a program that iterates through an array from the back to the front and prints out only the <b>odd</b> values.
```
Comment (bensussman) [inline lecture-loops.html]: can we make one of these front-to-back and the other back-to-front for diversity's sake?

Comment (mottaquikarim) [inline lecture-loops.html]: comin right up

**Comment by bensussman** [comment]
👍

**Pull request merged**